doc/CMakeLists.txt

```diff
@@ -0,0 +1,5 @@
+ADD_SUBDIRECTORY(man)
+
+INSTALL(FILES AUTHORS TYPE DOC RENAME AUTHORS.SF)
```

Comment:
https://github.com/vegastrike/Vega-Strike-Engine-Source/pull/416/checks?check_run_id=1812227168 failed b/c the `INSTALL(FILES..)` functionality used didn't exist until CMake 3.14, so I need to adjust to support the older functionality. Planning on leaving this version in but with a version check to swap out for the older style if needed. We can drop down to this once we can update to using CMake 3.14 or later.

Reply:
Asked the CMake community about it in their Discourse forum and got a really good answer. fc9044f makes the advised adjustment. Hopefully I got the IF blocks right.